src/views/SelectProduct.vue

```diff
@@ -323,13 +323,24 @@ export default defineComponent({
 </script>
 
 <style scoped>
-
 .section-grid {
-  grid-template-columns: repeat(auto-fill, 200px);
+  grid-auto-flow: column;
+  grid-auto-columns: 200px;
+  grid-template-columns: none;
+  width: 100vw;
+  overflow-x: scroll;
+  scroll-snap-type: x mandatory;
+}
+
+.section-grid > div {
+  scroll-snap-align: start;
```

Comment:
are we sure this is supposed to go on scrolling element? I think it should be on scroll container. Please double check and let me know.

Reply:
Yes Sir, Scroll snapping is used by setting the `scroll-snap-type` property on a container element and the `scroll-snap-align` property on elements inside it. When the container element is scrolled, it will snap to the child elements you’ve defined: https://css-tricks.com/practical-css-scroll-snapping/

If I take reference of commerce hub app, In that app also we have done the same thing : https://github.com/hotwax/commerce-hub/blob/00c5d26dd44a6774f723a2defc55936d8af67e55/src/views/Home.vue#L275